### ISSUE TITLE
New Nuxt Typescript Project Error

### ISSUE THREAD

**Issue by carlflor** [nuxt/typescript#377]
**Describe the bug**
I created a new project using:
```
$ yarn create nuxt-app <project-name>
```

**To Reproduce**
1. Clone repository 'git@github.com:carlflor/website.git'
2. Run 'yarn dev'
3. See error
```
 Property 'component' does not exist on type 'RouteConfig'
```

**Expected behavior**
error in the `.d.ts` file

**Screenshots**
![Screen Shot 2020-06-14 at 12 22 35 AM](https://user-images.githubusercontent.com/4861092/84573834-80470a00-add5-11ea-8b93-9657b1c5aed9.png)

I haven't done any changes since setting up the project

**Comment by kevinmarrec** [comment]
@carlflor A fix has been released in `@nuxt/types@0.7.8`, but the `CNA` (create-nuxt-app) is installing `0.7.5` cause of unupdated `@nuxt/typescrip-build` dependency, I'm gonna see with Nuxt team to fix it asap.

**Comment by kevinmarrec** [comment]
You can fix it for now by changing the version of `@nuxt/typescript-build` to `^1.0.0` and reinstall dependencies

**Comment by kevinmarrec** [comment]
@carlflor I released a fix on a `@nuxt/typescript-build@0.6.7`, all you need is to either clearing and installing again your dependencies, or create again the project. 

`@nuxt/typescript-build` will be correctly updated to `^1.0.0` when Nuxt 2.13 will be out.

**Comment by carlflor** [comment]
Thank you very much! Really helpful! :)
